ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-create-test.js

```diff
@@ -1007,29 +1007,6 @@ module('Acceptance | kv-v2 workflow | secret and version create', function (hook
     });
   });
 
-  module('secret-nested-creator persona', function (hooks) {
-    hooks.beforeEach(async function () {
-      const token = await runCmd(
-        tokenWithPolicyCmd(
-          `secret-nested-creator-${this.backend}`,
-          personas.secretNestedCreator(this.backend)
```

Comment:
Do we use this persona anywhere else, or can we clean it up since we don't have this popup option anymore?

Reply:
It's still being used in the delete workflow